### PR TITLE
home-page: fix search bar

### LINKED ISSUE #328: UI: Homepage search does not redirect to searchpage
Entering a query in the homepage searchbar and clicking the search button or pressing enter doesn't redirect to the search page

### PR DESCRIPTION
**Pull request by vlad-bm**
(Closes #328)

### CODE REVIEW

**Review by zzacharo — APPROVED** [review]

File: invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js [hunk 1]
```diff
@@ -15,39 +11,49 @@ import { default as config } from './config';
...
-                url: documentApi.url,
-              }}
-            >
-              <Container className="books-search-searchbar">
-                <SearchBar renderElement={this._renderSearchBar} />
-              </Container>
-            </ReactSearchKit>
+            <Container className="books-search-searchbar">
+              <Form
+                onSubmit={goTo(
```
Comment (equadon) [inline invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js:33]: Is `onSubmit` needed here when there is an `onClick` in the icon link?

**Review by equadon — APPROVED** [review]

File: invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js (hunk 1 above)
Comment (zzacharo) [reply invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js:33]: I think is for the enter event but still there is an event for that I think...

Comment (vlad-bm) [reply invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js:33]: Indeed, it is

Comment (equadon) [reply invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js:33]: Yes, you're right, I forgot about that!

**Pull request merged**